axelrod/strategies/axelrod_first.py

```diff
@@ -522,15 +523,15 @@ def __init__(self, alpha: float=0.05) -> None:
         """
         Parameters
         ----------
-        alpha, float
-            The significant level of pvalue from chi-squared test
-            0.05 by default according to literature
+        alpha: float
+            The significant level of p-value from chi-squared test
+            005 by default according to literature
         """
         super().__init__()
         self.alpha = alpha
-        self.opponent_is_random = None
```

Comment:
By the way, this was incorrect but the `Player__eq__` method was not catching it because it assumes that attributes that it doesn't find are `None`. The reset method was resetting ` self.random_opponent` instead of `opponent_is_random`.